source/NVDAObjects/window/excel.py

```diff
@@ -994,7 +994,7 @@ def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 		if formatConfig['reportFontName']:
 			formatField['font-name']=fontObj.name
 		if formatConfig['reportFontSize']:
-			formatField['font-size']=str(fontObj.size)
+			formatField['font-size']= _("%d pt") % int(fontObj.size)

```

Comment:
Translator comment is missing.
The checkPot does not complain about it probably because an other identical comment has a translator comment somewhere else. But IMO it's better to put the translators comment anyway.